lib/rspecq/queue.rb

```diff
@@ -70,10 +70,13 @@ class Queue
 
     attr_reader :redis
 
-    def initialize(build_id, worker_id, redis_host)
+    attr_reader :fail_fast
+
+    def initialize(build_id, worker_id, redis_host, fail_fast=nil)
```

Comment:
Since this argument is not strictly necessary when initializing a Queue, we can just drop it from this signature. This makes the method simpler (1 less argument) and we also get rid of the assignment below (`@fail_fast = fail_fast`).